VRCFaceTracking.Core/VRChat.cs

```diff
@@ -126,5 +126,5 @@ public static bool ForceEnableOsc()
         return wasOscForced;
     }
 
-    public static bool IsVrChatRunning() => Process.GetProcesses().Any(x => x.ProcessName == "VRChat");
+    public static bool IsVrChatRunning() => Process.GetProcesses().Any(x => x.ProcessName == (OperatingSystem.IsLinux() ? "VRChat.exe" : "VRChat"));
```

Comment:
This condition looks inverted - If we're on Linux, it'll look for `VRChat.exe` and `VRChat` on Windows.

```suggestion
    public static bool IsVrChatRunning() => Process.GetProcesses().Any(x => x.ProcessName == (OperatingSystem.IsWindows() ? "VRChat.exe" : "VRChat"));
```

Reply:
It's running the Windows VRChat under Proton so it shows up like this in the process list:
```
$ pgrep -l VRChat
1228141 VRChat.exe
```

I've verified this thru the debugger as well. I think I'm just gonna change it to work with either process name, regardless of platform.

Reply:
`Process.GetProcesses()` does not return processes with a trailing `.exe`? Looking at https://learn.microsoft.com/en-us/dotnet/api/system.diagnostics.process.getprocesses?view=net-7.0

Will need to verify on Windows

Reply:
So modified it so that it can match either process name: `VRChat` or `VRChat.exe`

Also made it do the same thing for all platforms, since I think it should cover all cases.